lib/uplink/project.go

```
@@ -150,6 +152,8 @@ func (p *Project) GetBucketInfo(ctx context.Context, bucket string) (b storj.Buc
 func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *EncryptionAccess) (b *Bucket, err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	attribution, _ := p.CheckBucketAttribution(ctx, bucketName)
```

don't ignore errors plz

---

added the error handling.